docs/runbook/formulare-support.md

```diff
@@ -1,155 +1,78 @@
+Formulár sa spracúva cez RabbitMQ od stavu `QUEUED` až po koncové stavy (`PROCESSING`, `FINISHED`).
 
-V ramci posielania formularu do ginisu mame dalsie moznosti stavov. Pre aktualne si kukni: [schema.prisma](https://github.com/bratislava/konto.bratislava.sk/blob/c0332af5d71159a0a42dd20ce9a386b2279bf912/nest-forms-backend/prisma/schema.prisma#L49) Kedze ginis nema pre nas nejake rozumne API, tak formular sa posiela do ginisu cez "klikacku" [ginis-automation](https://github.com/bratislava/ginis-automation) ktora vyuziva selenium a manualne tam robi kroky, ako keby ich robil pouzivatel. Bohuzial je to velmi nespolahlive, napriklad ked sa aktualizuje ginis, tak klikacka nie vzdy spravne funguje a musi sa tiez aktualizovat. Z tohto dovodu mame samostatnu rabbit mq ktora sluzi na menezement pridavania formularu do ginisu.
+### GINIS stavy:
 
-### Podme si popisat jednotlive stavy:
+- `CREATED` - vytvorený v queue
+- `RUNNING_REGISTER` / `REGISTERED` - registrácia cez podateľňu
+- `RUNNING_UPLOAD_ATTACHMENTS` / `ATTACHMENTS_UPLOADED` - upload príloh
+- `RUNNING_EDIT_SUBMISSION` / `SUBMISSION_EDITED` - úprava podania
+- `RUNNING_ASSIGN_SUBMISSION` / `SUBMISSION_ASSIGNED` - priradenie oddeleniu
+- `FINISHED` - dokončené
+- `ERROR_*` - rôzne chybové stavy
 
-- `CREATED` - formular bol vytvoreny a pridany do klikackovej queue
-- `RUNNING_REGISTER` - registrujeme formular cez podatelnu
-- `REGISTERED` - formular bol registrovany
-- `RUNNING_UPLOAD_ATTACHMENTS` - spusta sa upload priloh
-- `ATTACHMENTS_UPLOADED` - prilohy uploadnue
-- `RUNNING_EDIT_SUBMISSION` - upravuje sa podanie
-- `SUBMISSION_EDITED` - podanie upravene
-- `RUNNING_ASSIGN_SUBMISSION` - priraduje sa formular oddeleniu
-- `SUBMISSION_ASSIGNED` - odelenie priradene
-- `FINISHED` - skoncene
-- `ERROR_REGISTER` - vznikol error pri registracii podania
-- `ERROR_ATTACHMENT_UPLOAD` - vznikol error pri uploade
-- `ERROR_EDIT_SUBMISSION` - vznikol error pri editaci podania
-- `ERROR_ASSIGN_SUBMISSION` - vznikol error pri priradovani podania.
+## Riešenie problémov
 
-## Ako najst zaseknuty formular?
+### Kontrola stavu
 
-Tvoj den by mal zacat pozrenim si statusu formularov v [metabase](https://metabase.bratislava.sk/dashboard/11-forms-dashboard?date_filter=past7days) popripadne v channeli [#metabase-forms](https://inovaciebratislava.slack.com/archives/C07HQGBPNDA)
+1. Kontrolovať [metabase](https://metabase.bratislava.sk/dashboard/11-forms-dashboard?date_filter=past7days) alebo #metabase-forms
+2. Sledovať hlavne stavy `DELIVERED_GINIS`, `DELIVERED_NASES`, `SENDING_TO_SHAREPOINT`, v prípade daní `SENDING_TO_NASES`
```

Comment:
@lukasbabula  I've added a couple states here, otherwise left untouched - but I suspect this part needs more info ? please expand as you see fit.